### PR TITLE
UI for movie-db 

### PR DESCRIPTION
**Pull request by ADITIRAI290698**
[Discussion](https://github.com/orgs/FifthTry/discussions/90)

### CODE REVIEW

File: dj/ui/FPM.ftd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- import: fpm
+
+-- fpm.package: movie-db
+base: https://raw.githubusercontent.com/FifthTry/movie-db
+
+-- fpm.dependency: fifthtry.github.io/doc-site as ds
```
Comment (amitu) [inline dj/ui/FPM.ftd]: We should not use doc-site for this project. Create a custom page please. Feel free to use dependencies of doc-site, eg header, typography etc, but create custom UI.

File: dj/ui/add_movie.ftd [hunk 2]
```diff
@@ -61,6 +94,12 @@ optional string list-desc3:
 optional string list-desc4:
 optional string list-desc5:
 optional string list-desc6:
+optional string list-desc7:
+optional string list-desc8:
+optional string list-desc9:
+optional string list-desc10:
+optional string list-desc11:
+optional string list-desc12:
```
Comment (amitu) [inline dj/ui/add_movie.ftd]: Why is this not a list?

Comment (ADITIRAI290698) [reply dj/ui/add_movie.ftd]: will make the change.

**Comment by amitu** [comment]
@ADITIRAI290698, @AviralVerma13 ensure this PR is merged today. You do not have to wait for a feature to be done before you merge the PR. You block merging a PR only if you believe there would an actual problem if the PR is merged.

**Pull request merged**